core/src/avm2/globals.rs

```diff
@@ -89,13 +89,16 @@ pub struct SystemClasses<'gc> {
     pub qname: ClassObject<'gc>,
     pub sharedobject: ClassObject<'gc>,
     pub mouseevent: ClassObject<'gc>,
+    pub progressevent: ClassObject<'gc>,
     pub textevent: ClassObject<'gc>,
     pub errorevent: ClassObject<'gc>,
     pub ioerrorevent: ClassObject<'gc>,
     pub securityerrorevent: ClassObject<'gc>,
     pub transform: ClassObject<'gc>,
     pub colortransform: ClassObject<'gc>,
     pub matrix: ClassObject<'gc>,
+    pub illegaloperationerror: ClassObject<'gc>,
+    pub event_dispatcher: ClassObject<'gc>,
```

Comment:
Should this be `eventdispatcher` to match the naming convention?

(Separate from this PR, should we use `snake_case` the naming convention for all of these eventually?)

Reply:
I think snake case would be better in the long run, but I'm fine with changing this to `eventdispatcher` for now.